etc/generate-envoy-config/pachyderm-services.libsonnet

```diff
@@ -147,6 +147,16 @@
           timeout: '604800s',
         },
       },
+      {
```

Comment:
I think there are extra files in the PR.  I think to get this into master, you want to start fresh, add those two lines to the proto builder thingie (skip grpc gateway, just add the ts api generator stuff), then make proto.

Reply:
Right, I guess I don't actually need Zartaj's branch merged before this one goes in, since it is just generating types. Will update.